examples/on_gke_end_to_end/variables.tf

```diff
@@ -18,10 +18,41 @@ variable "credentials_path" {
   description = "Path to service account json"
 }
 
+variable "config_validator_enabled" {
+  description = "Config Validator scanner enabled."
+  default     = false
```

Comment:
Should add type = bool here

Reply:
given that Terraform [converts types](https://www.terraform.io/docs/configuration/expressions.html#type-conversion) is this necessary?

Reply:
Then we should take out the type = bool for the other variable. Just want to be consistent!

Reply:
Actually we need to.  It won't render in the README as a bool if the type isn't explicit here.